.github/workflows/release-docker.yml

```diff
@@ -68,10 +68,18 @@ jobs:
       - name: Get docker build arg
         id: build_arg
         run: |
-          BUILD_ARG_NAME=$(echo ${{ inputs.package }} | sed 's/\-/_/g')
-          BUILD_ARG_VALUE=$(echo ${{ inputs.package }} | sed 's/postgres/core/g')
-          echo "name=$BUILD_ARG_NAME" >> $GITHUB_OUTPUT
-          echo "value=$BUILD_ARG_VALUE" >> $GITHUB_OUTPUT
+          repo_arg=$(echo ${{ inputs.package }} | sed 's/\-/_/g')
+          if [[ ${{ inputs.version_number }} < 1.8.0 ]];
```

Comment:
I'm not sure if this will work when we get to 1.10.0.

Reply:
It won't. You need something:
```vercomp () {
    if [[ $1 == $2 ]]
    then
        return 0
    fi
    local IFS=.
    local i ver1=($1) ver2=($2)
    # fill empty fields in ver1 with zeros
    for ((i=${#ver1[@]}; i<${#ver2[@]}; i++))
    do
        ver1[i]=0
    done
    for ((i=0; i<${#ver1[@]}; i++))
    do
        if [[ -z ${ver2[i]} ]]
        then
            # fill empty fields in ver2 with zeros
            ver2[i]=0
        fi
        if ((10#${ver1[i]} > 10#${ver2[i]}))
        then
            return 1
        fi
        if ((10#${ver1[i]} < 10#${ver2[i]}))
        then
            return 2
        fi
    done
    return 0
}
```

Reply:
you could also mid build curl and run this: https://gist.github.com/Ariel-Rodriguez/9e3c2163f4644d7a389759b224bfe7f3#file-test-sh-L11

Reply:
Yeah, I figured that was too easy.

Reply:
I think it's going to be easier to just revert the change, keep this process for 1.7 and prior, and then handle the new releases in the `dbt-postgres` repo.